pipelines/managed/fbc-release/README.md

```diff
@@ -20,6 +20,9 @@ Tekton release pipeline to interact with FBC Pipeline
 | taskGitUrl                      | The url to the git repo where the release-service-catalog tasks to be used are stored                                                      | Yes      | https://github.com/konflux-ci/release-service-catalog.git |
 | taskGitRevision                 | The revision in the taskGitUrl repo to be used                                                                                             | No       | -                                                         |
 
+## Changes in 4.7.0
+* update to specify new required params for task update-ocp-tag
```

Comment:
Just mentioning - you have a lot of open PRs right now doing this for various tasks in fbc-release. I would be fine with keeping it at 4.7.0 and just adding another bulletpoint to the list here when one merges and you go to rebase the next and fix conflicts (we are doing away with change logs and versions eventually anyways)